src/lib/sdc/usb-key.sh

```diff
@@ -133,6 +135,45 @@ function unmount_usb_key()
 	umount "$mnt"
 }
 
+# Evil twins of the {,un}mount_usb_key functions.  These are specific
+# to ISO/hsfs (aka. {C,DV,B}D-ROM) disks.  We may be able to factor-out
+# even more common bits from the usb_key functions, but not today.
+
+function mount_ISO
```

Comment:
nit: lowercase ISO? from a quick glance all function names are lowercase

Reply:
ISO in all caps is easier to grep.  Not sure it's worth it.